elements/lisk-chain/src/block.ts

```diff
@@ -45,7 +51,7 @@ export class Block {
 		);
 	}
 
-	public static fromJSON(value: Record<string, unknown>): Block {
+	public static fromJSON(value: BlockJSON): Block {
```

Comment:
I understand that we use `BlockJSON` type as the return type of `toJSON` method. I don't understand why we use a type with non-unknown properties and also have type guards in the method code. Why do we do that?